components/dropdown/dropdown-context-menu.js

```diff
@@ -0,0 +1,67 @@
+import '../button/button-icon.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { VisibleOnAncestorMixin, visibleOnAncestorStyles } from '../../mixins/visible-on-ancestor-mixin.js';
+import { DropdownOpenerMixin } from './dropdown-opener-mixin.js';
+import { dropdownOpenerStyles } from './dropdown-opener-styles.js';
+
+class DropdownContextMenu extends DropdownOpenerMixin(VisibleOnAncestorMixin(LitElement)) {
+
+	static get properties() {
+		return {
+			/**
+			 * Label for the context-menu button (required for accessibility).
+			 */
+			text: {
+				type: String
+			},
+			translucent: {
+				type: Boolean
+			},
+		};
+	}
+
+	static get styles() {
+		return [dropdownOpenerStyles, visibleOnAncestorStyles, css`
+			:host {
+				display: inline-block;
+			}
+		`];
+	}
+	constructor() {
+		super();
+		this.translucent = false;
+	}
+
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+
+		this.addEventListener('d2l-dropdown-open', this._onOpen);
+		this.addEventListener('d2l-dropdown-close', this._onClose);
+	}
+
+	render() {
+		return html`
+			<d2l-button-icon aria-label=${this.text} ?disabled=${this.disabled} icon="tier1:chevron-down" text=${this.text} ?translucent=${this.translucent}>
+			</d2l-button-icon>
+			<slot></slot>
+		`;
+	}
+
+	/**
+	 * Gets the "d2l-button-icon" opener element (required by dropdown-opener-mixin).
+	 * @return {HTMLElement}
+	 */
+	getOpenerElement() {
+		return this.shadowRoot.querySelector('d2l-button-icon');
+	}
+
+	_onClose() {
+		this.removeAttribute('opened');

```

Comment:
Do you know what the purpose of these handlers is?  Looks like the "more" opener has them too.

Reply:
I couldn't find anywhere they're being used but I can't 100% rule out cases that might be using `getAttribute("opened")` or referencing `opened` in an attribute selector that doesn't reference the `d2l-dropdown-context-menu` element explicitly since they're somewhat difficult to search for.

Reply:
Looks like it was used a long time ago.  The styles were removed here, and it looks like perhaps we should have removed these handlers at the same time... 🤔 

https://github.com/BrightspaceUI/dropdown/commit/2a08a6aa20ffabd4e4e5ded1812d4c894a29ebda#diff-abcef24286f2e79076d2e11d242b7528L40

I'd be keen to remove them if possible.

Reply:
Sounds good to me, I removed it! With having found where it was originally used and not finding anywhere else its referenced it should be relatively low risk to remove it.